### ISSUE TITLE
"Enable window peeking" causes peeked windows to temporarily replace GNOME Desktop background

### ISSUE THREAD

**Issue by elliotwutingfeng** [home-sweet-gnome/dash-to-panel#928]
**Describe the bug**
When the "Enable window peeking" feature (under Behavior>Show window previews on hover>Window preview options) is enabled, hovering the mouse cursor over the window preview of a minimized application can sometimes cause the peeked window of that application and/or other open applications to be "imprinted" on the GNOME desktop background. To get rid of the "imprinting" temporarily, the affected windows have to be maximised and minimised.

**Linux distribution and version**
Manjaro 19.02

**GNOME Shell version**
GNOME Shell 3.36.0

**Dash-to-Panel version**
31

**Where was Dash-to-Panel installed from?**
GNOME extensions website

**Screenshots / Video captures**
![bug](https://user-images.githubusercontent.com/30223404/78272632-5c975600-7540-11ea-8f56-c91527d39eb2.png)

**Additional Info**
nil

**Comment by charlesg99** [comment]
Thank you for the report elliotwutingfeng! I could reproduce the issue, it is a side effect of a [mutter bug](https://gitlab.gnome.org/GNOME/mutter/issues/836#note_743720). I just added a workaround for it (big thanks to @Noobsai) on the master branch, could you please install it and see if that works on your end? Thanks!

**Comment by jderose9** [comment]
@charlesg99 While trying to reproduce this, I noticed that window peeking opacity seems to be broken since v27. Do you observe that? (All windows aside from the peeked one disappear)

**Comment by charlesg99** [comment]
@jderose9 Good timing :)

**Comment by jderose9** [comment]
so..... v33? :)

**Comment by charlesg99** [comment]
Lol, if you want to! We're on a roll!

**Comment by elliotwutingfeng** [comment]
@charlesg99 Problem appears to be gone now, the window peeking opacity is also working properly. Thanks!

**Comment by jderose9** [comment]
I'm seeing some oddities with the new commits on Fedora 31/GS 3.34. 
- Firefox opacity doesn't seem to change (it's always fully visible)
- Chromium is flickering/flashing when it's in the background during a peek
- I once got stuck in a weird state where the opacity wasn't changing on any pixels within the borders of one the background windows:

![Screenshot from 2020-04-02 14-25-26](https://user-images.githubusercontent.com/2118702/78286538-eddddc80-74ee-11ea-8247-e5a23d2b485b.png)

**Comment by charlesg99** [comment]
Hum I guess the workaround for the animation only works on 3.36+ :( That makes sense as I tried the proposed mutter workaround some time ago and dismissed it as it wasn't working properly (got the same as you mentioned), but I was still on 3.34 then... My memory seems to be failing! 

Anyway I pushed another fix, so only 3.34 is condemned of having no window fade animations. Still happy that it is working on 3.36 though:

![Peek 2020-04-02 15-01](https://user-images.githubusercontent.com/33044886/78291516-88d8b580-74f3-11ea-8cca-55029d7e5ad7.gif)

Thanks!
